docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -49,6 +49,20 @@ The <<search-template,search template>> associated with this search application.
 - The template may be modified with subsequent <<put-search-application,put search application>> requests.
 - If no template is specified when creating a search application, or if a template is removed from a search application, we use the <<query-string-query-ex-request,query_string>> defined in the template examples as a default.
 - This template will be used by the <<search-application-search,search application search>> API to execute searches.
+- The template accepts an optional `dictionary` parameter which defines a JSON schema used for validating parameters sent to the <<search-application-search,search application search>> API
```

Comment:
Is it worth adding a warning here that, while the dictionary is optional, not specifying it will not validate input into the template? Alternately if we don't want to word it that strongly, perhaps a sentence about why users may want to use the dictionary?